products/developments/housing_by_cb/metadata.yml

```diff
@@ -0,0 +1,219 @@
+id: housing_database_by_2020_census_block
+
+attributes:
+  description: |-
+    Net change in housing units arising from new buildings, demolitions, or alterations for NYC Census Blocks since 2010.  The NYC Department of City Planning's (DCP) <a href="https://www1.nyc.gov/site/planning/data-maps/open-data/dwn-housing-database.page#housingdevelopmentproject">Housing Database</a> provide the 2010 census count of housing units, the net change in Class A housing units since the census, and the count of units pending completion for commonly used political and statistical boundaries. These tables are aggregated from the DCP Housing Database, which is derived from Department of Buildings (DOB)-approved housing construction and demolition jobs filed or completed in NYC since January 1, 2010. Net housing unit change is calculated as the sum of all three construction job types that add or remove residential units: new buildings, major alterations, and demolitions, and can be used to determine the change in legal housing units across time and space.
+    {{ bytes_previous_versions_link }}
+  display_name: Housing Database by 2020 Census Block
+  each_row_is_a: census block
+  tags:
+  - housing
+  - house
+  - building
+  - development
+  - dwelling unit
+  - certificate of occupancy
+  - co
+  - permit
+  - job application
+  - filing
+  - net units
+  - unit change
+  - pipeline
+  - construction
+  - census block
+  - census tract
+  - nta
+  - puma
+  - city council district
+  - community district
+
+assembly: []
+
+custom: {}
+
+destinations:
```

Comment:
If we want to make this a one-click deployment type of situation, we can do that by adding a Bytes destination (example [here](https://github.com/NYCPlanning/product-metadata/blob/7f044f6af533d9adb62c98e09966a7924de96a5e/products/lion/pseudo_lots/metadata.yml#L26)), which will allow us to package up all the files straight from Bytes.

Reply:
In those cases, I usually go to Bytes and infer the dataset/attachment url scheme, which often includes the product release version. In this case, I'd look [here] on bytes. Looks like the shapefile has a scheme of https://s-media.nyc.gov/agencies/dcp/assets/files/zip/data-tools/bytes/nychdb_cblock_{{ version }}_shp.zip